### ISSUE TITLE
explain parameters of delete function

### ISSUE THREAD

**Issue by thawn** [swcarpentry/python-novice-inflammation#572]
Lesson 01 the excercise "Stacking Arrays":
the second solution that uses `numpy.delete` just mentions the delete function without explaining how it works:

> An alternative way to achieve the same result is to use Numpy’s delete function to remove the second column of A.

It would be nice to explain how the parameters work (i.e. the first parameter is the array, the second the dimension (row/column) and the third is the index of the dimension that should be deleted (do index ranges work here?)

**Comment by annefou** [comment]
Yes or at least to explain again how to get help to get info on the list of possible arguments. To be honest, I don't usually teach this part as this first episode is so long.
